deploy/003-deploy-mock-tokens.ts

```diff
@@ -0,0 +1,34 @@
+import { ethers } from "hardhat";
+import { DeployResult } from "hardhat-deploy/dist/types";
+import { DeployFunction } from "hardhat-deploy/types";
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import { convertToUnit } from "../helpers/utils";
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { deployments, getNamedAccounts }: any = hre;
+  const { deploy } = deployments;
+  const { deployer, proxyAdmin } = await getNamedAccounts();
+  //=======================
+  // DEPLOY MOCK TOKENS
+  //========================
+  await deploy("MockBNX", {
+    from: deployer,
+    contract: "MockToken",
+    args: ["BinaryX", "BNX", 18],
+    log: true,
+    autoMine: true, // speed up deployment on local network (ganache, hardhat), no effect on live networks
+  });
+
+  await deploy("MockBSW", {
+    from: deployer,
+    contract: "MockToken",
+    args: ["Biswap", "BSW", 18],
+    log: true,
+    autoMine: true,
+  });
+
+};
+
+func.tags = ["MockTokens"];
```

Comment:
Should we add a skip here so it only runs on local / test networks?

Reply:
yeah I will add here as well